Frends.FixedWidthFlatFile.ConvertToJSON/Frends.FixedWidthFlatFile.ConvertToJSON/Definitions/Input.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+
+namespace Frends.FixedWidthFlatFile.ConvertToJSON.Definitions
+{
+    public class Input
+    {
+        /// <summary>
+        /// Fixed width flat file content
+        /// </summary>
+        [DisplayFormat(DataFormatString = "Expression")]
```

Comment:
You can remove this line, since the expression should not be used as default input type. Without this line, Frends should show the input options as a list option, since the property data type is a list.